configure.ac

```diff
@@ -2408,3 +2408,8 @@ AC_OUTPUT
 
 final_message="$final_message\n  prefix:\t$prefix\n$dashes"
 echo -e "$final_message\n"
+echo "#------------------------------------ WARNING ----------------------------------------#"
+echo "The autotools based build system is deprecated and will be removed in the next version!"
```

Comment:
No strong opinion, but I prefer it as is. That way we don't have to change it, if we need to postpone removal for some reason...